ocrs_wrapper.py

```
@@ -0,0 +1,25 @@
+
+
+def run_ocrs_cli(args):
+    """
+    Run ocrs-cli.exe with the given arguments.
+    Args:
+        args (list): List of command-line arguments (excluding the executable).
+    Returns:
+        stdout, stderr: Output and error from the process.
+    """
+    exe = "ocrs-cli.exe"  # Change to "ocrs-clie.exe" or "ocrs.exe" as needed
+    cmd = [exe] + args
```

Ruff (0.13.1)

Consider `[exe, *args]` instead of concatenation

Replace with `[exe, *args]`

(RUF005)